src/models/tiles/geometry/geometry-content.ts

```diff
@@ -1151,3 +1152,12 @@ export type GeometryContentModelType = Instance<typeof GeometryContentModel>;
 export type GeometryContentSnapshotType = SnapshotIn<typeof GeometryContentModel>;
 
 export type GeometryMigratedContent = [GeometryContentModelType, { title: string }];
+
+export function updateGeometryContentWithNewSharedModelIds(
+  content: GeometryContentModelType,
+  sharedDataSetEntries: PartialSharedModelEntry[],
+  updatedSharedModelMap: Record<string, UpdatedSharedDataSetIds>
+) {
+  // Geometry content has no references to shared dataset ids, so we can just return the unmodified content
+  return content;
+}
```

Comment:
Is this necessary? Seems like if we used something like the `TileContentInfo` registration system I mentioned above, it would make sense to assume that any tile that didn't register a function should use the unmodified content and there wouldn't be any need for such tiles to explicitly implement the null transform.